.github/workflows/release.yml

```diff
@@ -134,7 +134,8 @@ jobs:
             exit 1
```

Comment:
In this code change, the script is modified to save the sanitized JIRA issue key to a temporary file called temp.env instead of directly appending it to the $GITHUB_ENV file. After saving the key to temp.env, the content of temp.env is then appended to $GITHUB_ENV using the `cat` command. This change separates the process of generating the sanitized key from adding it to the environment variable, making the code more modular and easier to understand.